schemas/iso19115-3.2018/src/main/plugin/iso19115-3.2018/convert/fromJsonOpenDataSoft.xsl

```diff
@@ -546,6 +550,12 @@
 
     <xsl:template name="dataLink">
       <xsl:param name="format" />
+      <xsl:param name="fields" />
+      <xsl:variable name="description">
+        <xsl:for-each select="$fields/fields">
+          <xsl:value-of select="concat('- *', label, '* : ', name, '[', type, ']\n')" />
+        </xsl:for-each>
+      </xsl:variable>
```

Comment:
I don't think this is necessary, the fields are already described here: https://github.com/geonetwork/core-geonetwork/blob/34cd3d5f57d91f3a738c7f8fe1eafcd1f0259477/schemas/iso19115-3.2018/src/main/plugin/iso19115-3.2018/convert/fromJsonOpenDataSoft.xsl#L414-L438